### PR TITLE
2018 changes

### PR DESCRIPTION
**Pull request by sivakumar-kailasam**

### CODE REVIEW

**Review by ladyleet — APPROVED** [review]
Looks great @sivakumar-kailasam! Thx for catching some of the things I didn't think about as well like Ben's new job title. ;)

**Comment by sivakumar-kailasam** [comment]
👍

File: app/templates/index.hbs [hunk 1]
```diff
@@ -1,8 +1,10 @@
 <section class="container about">
   <div class="row">
     <div class="content-box col-md-7">
-      <h2>Managing Async With RxJS 5: Course Overview</h2>
-      <p>This is a 3 part course complete with pre-course videos, an interactive workshop day, and post workshop support.</p>
+      <h2>Managing Async With RxJS 5.5+: Course Overview</h2>
+      <p>This is a 2 part course complete with pre-course videos and an interactive pre-recorded 8 hour workshop.</p>
```
Comment (ladyleet) [inline app/templates/index.hbs:5]: @sivakumar-kailasam this says pre-recorded but does it describe all of them? should it say interactive workshop day?

Comment (sivakumar-kailasam) [reply app/templates/index.hbs:5]: I took this from the eventbrite description. If you give a sentence in mind, I can update it.

**Pull request merged**